### PR TITLE
Bump inline-style-prefixer

### LINKED ISSUE #604: The `display` property is not being applied to a lot of elements in IE10
This is a very weird one I stumbled on today. Apparently, Radium is not applying (or maybe IE10 is dropping) the `display` styles to the element's inline styles. Everything else seems to be working.

I'll try to get a repro on this, maybe tomorrow...

### PR DESCRIPTION
**Pull request by alexlande**
Resolves #604

/cc @ianobermiller @tptee @rofrischmann

### CODE REVIEW

File: src/__tests__/style-component-test.js [hunk 1]
```diff
@@ -63,7 +63,6 @@ describe('<Style> component', () => {
       div{
         transform:rotate(90);
         -webkit-transform:rotate(90);
-        -moz-transform:rotate(90);
```
Comment (alexlande) [inline src/__tests__/style-component-test.js]: As much as it pains me, I guess this counts as a breaking change.

**Comment by alexlande** [comment]
@ianobermiller: Any objection to me merging this one and publishing a release today?

File: src/__tests__/style-component-test.js (hunk 1 above)
Comment (ianobermiller) [inline src/__tests__/style-component-test.js]: Oh, because latests Firefox no longer requires it.

**Comment by ianobermiller** [comment]
Not at all, please do. There are some other fixes that people have been waiting a while for. I apologize that I haven't had time to spend on Radium! Looking to list our house in two weeks :S

**Comment by alexlande** [comment]
Exciting! No worries at all :+1:

**Pull request merged**